src/components/organisms/Form/FormCheckboxField/FormCheckboxField.tsx

```diff
@@ -1,25 +1,19 @@
-import React, { FC, ChangeEvent } from 'react';
+import React from 'react';
+import { useField, FieldHookConfig } from 'formik';
 import CheckboxField, { ICheckboxFieldProps } from '../../../molecules/CheckboxField/CheckboxField';
-import { useFieldContext } from '../hooks';
 
-interface IFormCheckboxFieldProps extends ICheckboxFieldProps {
-  name: string;
-}
+export type TFormCheckboxFieldProps = Pick<FieldHookConfig<boolean>, 'validate' | 'name'> & ICheckboxFieldProps;
```

Comment:
Isn't hungarian notation deprecated now?

Reply:
In this repo so far we use both ( _hungarian_ vs _no-hunagarian_ ) according to personal preference and so far we haven't cared much to clean-up 💥 🧹

I agree is better to be consistent with it, however we haven't seen any productivity issue with it and hence has become a priority to refactor 😁